src/quo2/core.cljs

```diff
@@ -113,6 +114,7 @@
 (def checkbox quo2.components.selectors.selectors.view/checkbox)
 (def filter quo2.components.selectors.filter.view/view)
 (def skeleton quo2.components.loaders.skeleton/skeleton)
+(def static-skeleton quo2.components.loaders.skeleton.view/view)
```

Comment:
could you make a loaders section and move it to that? same for other `loaders` in this file!